src/Util.ts

```diff
@@ -7,6 +7,6 @@ export type Constructor<T = EventEmitter> = new (...args: any[]) => T;
  * @param {Object} obj
  * @return {boolean}
  */
-export function isIterable(obj: Iterable<unknown>) {
+export function isIterable(obj: Iterable<unknown>):boolean {
```

Comment:
*nit*: One convention throughout the codebase is to add a space after the colon for return types such as `: boolean` -- we likely need to set up a TSLint if it's not already done.